lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -400,26 +400,54 @@ export class BrowserCacheManager extends CacheManager {
      * Gets the active account
      */
     getActiveAccount(): AccountInfo | null {
-        const activeAccountIdKey = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
-        const activeAccountId = this.browserStorage.getItem(activeAccountIdKey);
-        if (!activeAccountId) {
+        const activeAccountKeyHomeLocalIds = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT_HOME_LOCAL_IDS);
+        const activeAccountValueHomeLocalIds = this.getItem(activeAccountKeyHomeLocalIds);
+        if (!activeAccountValueHomeLocalIds) { 
+            // if new active account cache type isn't found, it's an old version, so look for that instead
+            const activeAccountKeyLocal = this.generateCacheKey(PersistentCacheKeys.ACTIVE_ACCOUNT);
+            const activeAccountValueLocal = this.getItem(activeAccountKeyLocal);
+            if(!activeAccountValueLocal)
```

Comment:
@shylasummers nit: JS code in general (as well as this code base) follows the [K&R_style](https://en.wikipedia.org/wiki/Indentation_style#K&R_style), can we do the same here and elsewhere?

```javascript
            if (!activeAccountValueLocal) {
                 // ...
            }
```

Reply:
Fixed! Thanks for letting me know